docs/standard/serialization/system-text-json/polymorphism.md

```diff
@@ -288,6 +288,9 @@ WeatherForecastBase value = JsonSerializer.Deserialize<WeatherForecastBase>(json
 Console.WriteLine(value is WeatherForecastWithCity); // True
 ```
 
+> [!NOTE]
+> The type discriminator must be the first property in the JSON for deserialization to succeed.
```

Comment:
This is true for any metadata property that STJ emits, it doesn't strictly speaking need to be the first property, however all metadata properties need to be placed before any regular properties.
```suggestion
> The type discriminator must be placed at the start of the JSON object, grouped together with other metadata properties like `$id` and `$ref`.
```